sdk/storage/azblob/README.md

```diff
@@ -0,0 +1,224 @@
+# Azure Storage Blob SDK for Go
+
+## Introduction
+
+The Microsoft Azure Storage SDK for Go allows you to build applications that takes advantage of Azure's scalable cloud
+storage. This SDK replaces the previously previewed [azblob package](https://github.com/azure/azure-storage-blob-go) in the separate repository.
+
+## Prerequisites
+
+* Go versions 1.16 or higher. If you don't already have it, install [the Go distribution](https://golang.org/dl/)
+* You must have an [Azure storage account][azure_storage_account]
+
+## Getting Started
+
+* Install the Azure blob storage client library for Go with `go get`:
+  ```bash
+  go get github.com/Azure/azure-sdk-for-go/sdk/storage/azblob
+  ```
+* Import SDK in your code:
+  ```golang
+  import "github.com/Azure/azure-sdk-for-go/sdk/storage/azblob"
+  ```
+
+## Code Samples
+
+`azblob` allows you to interact with three types of resources
+
+* The storage accounts.
+* The containers within those storage accounts.
+* The blobs (block blob/ page blob/ append blob) within those containers.
+
+To interact with these resources, start by creating the instances of each type
```

Comment:
Either remove this sentence or provide links on how to "create the instances of each type"